src/components/dialogs/StandardDialog.svelte

```diff
@@ -55,7 +55,7 @@
   });
 
   const keyListener = (event: KeyboardEvent) => {
-    if (event.key === 'Escape') {
+    if (event.key === 'Escape' && isOpen) {
       onCloseDialog();
     }
   };
```

Comment:
It's so unexpected how the key listener is listening even when the dialog is not opened. I guess it's because the conditional that determines whether the dialog is being rendered is in this component itself so regardless of the state of the dialog this whole script runs. Yikes, didn't think of that before